File: SDKs/Aspose.Words_Cloud_SDK_for_CSharp/src/Com/Aspose/Words/Model/SaveResult.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class SaveResult {
    public Link SourceDocument { get; set; }

    public Link DestDocument { get; set; }

    public List<Link> AdditionalItems { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class SaveResult {\n");
      sb.Append("  SourceDocument: ").Append(SourceDocument).Append("\n");
      sb.Append("  DestDocument: ").Append(DestDocument).Append("\n");
      sb.Append("  AdditionalItems: ").Append(AdditionalItems).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: Examples/Android/aspose-cloud-words-android/src/main/java/com/aspose/words/model/SaveResult.java
package com.aspose.words.model;

import java.util.*;
import com.aspose.words.model.Link;
public class SaveResult {
  private Link SourceDocument = null;
  private Link DestDocument = null;
  private List<Link> AdditionalItems = new ArrayList<Link>();
  /**
	 * getSourceDocument
	 * Gets Link
	 * @return SourceDocument
	 */
  public Link getSourceDocument() {
    return SourceDocument;
  }

	/**
	 * setSourceDocument
	 * Sets Link
	 * @param SourceDocument Link
	 */
  public void setSourceDocument(Link SourceDocument) {
    this.SourceDocument = SourceDocument;
  }

  /**
	 * getDestDocument
	 * Gets Link
	 * @return DestDocument
	 */
  public Link getDestDocument() {
    return DestDocument;
  }

	/**
	 * setDestDocument
	 * Sets Link
	 * @param DestDocument Link
	 */
  public void setDestDocument(Link DestDocument) {
    this.DestDocument = DestDocument;
  }

  /**
	 * getAdditionalItems
	 * Gets List<Link>
	 * @return AdditionalItems
	 */
  public List<Link> getAdditionalItems() {
    return AdditionalItems;
  }

	/**
	 * setAdditionalItems
	 * Sets List<Link>
	 * @param AdditionalItems List<Link>
	 */
  public void setAdditionalItems(List<Link> AdditionalItems) {
    this.AdditionalItems = AdditionalItems;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class SaveResult {\n");
    sb.append("  SourceDocument: ").append(SourceDocument).append("\n");
    sb.append("  DestDocument: ").append(DestDocument).append("\n");
    sb.append("  AdditionalItems: ").append(AdditionalItems).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

